lib/logstash/outputs/logstash.rb

```diff
@@ -119,6 +135,17 @@ def inner_http_output_options
     end
   end
 
+  def construct_host_uri
+    logger.warn("Multiple `hosts` are provided but single is supported, using the first.") if @hosts.size > 1
```

Comment:
I think it may make sense to print the host that's being used here. e.g:

```suggestion
    logger.warn("Multiple `hosts` are provided but single is currently supported, using the first (`#{ssl_config_name}`).") if @hosts.size > 1
```

Reply:
Sorry, it is not required, removed this log and also `list => true` option.